src/views/components/define-signatory-info/signatory-details.njk

```diff
@@ -1,20 +1,19 @@
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/input/macro.njk"  import govukInput %}
 
-{% set signatoryId = (signatory.id | lower) %}
-{% set fieldsetId = "signatory_" + signatoryId %}
+{% set fieldsetId = "signatory" + formSuffix %}
 
-{% set isSigningName = "isSigning_" + signatoryId %}
-{% set isSigningId = "is-signing_" + signatoryId %}
+{% set isSigningName = "isSigning" + formSuffix %}
```

Comment:
A formSuffix is now provided by the parent template i.e. change-details or define-signatory-info. This makes this template reusable in multiple places.